src/modules/tokens/models/esdtToken.model.ts

```diff
@@ -23,6 +23,10 @@ export class EsdtToken implements IEsdtToken {
     derivedEGLD: string;
     price?: string;
     previous24hPrice?: string;
+    previous7dPrice?: string;
+    priceChange24h?: string;
```

Comment:
no need for this field. It can be computed on frontend

Reply:
removed it